include/swift/AST/DiagnosticsSema.def

```diff
@@ -5455,6 +5455,12 @@ ERROR(distributed_actor_isolated_non_self_reference,none,
       "distributed actor-isolated %kind0 can not be accessed from a "
       "nonisolated context",
       (const ValueDecl *))
+NOTE(note_consider_method_for_isolated_property_mutation,none,
+     "consider declaring an %0 %1 method to perform the mutation",
+     (const NominalTypeDecl *, ActorIsolation))
+NOTE(note_consider_method_for_global_actor_isolated_property_mutation,none, // TODO: de-duplicate with note_consider_method_for_isolated_property_mutation
```

Comment:
How about phrasing this as

```
consider declaring an isolated method on %0 to perform the mutation
```

where `%0` is just a `Type`, which works for both kinds of actor isolation?

Reply:
That sounds good :) easier to make that diagnostic too, thanks